roles/vpn/templates/etc_openvpn_server.conf.j2

```diff
@@ -190,6 +190,7 @@ ifconfig-pool-persist ipp.txt
 ;push "redirect-gateway def1 bypass-dhcp"
 push "redirect-gateway def1"
 push "dhcp-option DNS 10.8.0.1"
+push "dhcp-option DNS 8.8.8.8"
```

Comment:
I use pure openvpn without additional dns server...
I think many installations does not contain dns server. Especially note that your etc_dnsmasq.conf does not contain any necessary options. In my opinion it should be moved to additional role with variables and flexible config template.